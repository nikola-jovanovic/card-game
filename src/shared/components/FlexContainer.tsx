import styled from 'styled-components'

const FlexContainer = styled.div`
  display: flex;
  flex: 1;
`

export default FlexContainer