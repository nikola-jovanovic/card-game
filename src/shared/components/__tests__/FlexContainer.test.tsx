import { render } from '@testing-library/react'
import React from 'react'

import FlexContainer from '../FlexContainer'

describe('FlexContainer', () => {
  it('render', () => {
    const { container } = render(<FlexContainer>click me</FlexContainer>)

    expect(container).toMatchSnapshot()
  })
})