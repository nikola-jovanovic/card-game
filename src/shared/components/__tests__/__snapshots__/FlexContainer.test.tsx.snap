// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlexContainer render 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

<div>
  <div
    class="c0"
  >
    click me
  </div>
</div>
`;
